### PR TITLE
Display ID in changes and problems like in tickets; fixes #2338

### LINKED ISSUE #2338: Difference in display of IDs between ticket and change
When editing a ticket, in the title we have the information of the id of the ticket.

ticket

However, for changes / problems, ..., the information is not present. 
change

We must activate the "Show GLPI ID" option.

Would it be possible to always have the id in the title for changes and problems even if the "Show GLPI ID" option is not enabled?

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2338

### CODE REVIEW

File: inc/change.class.php [hunk 1]
```diff
@@ -636,9 +636,17 @@ function showForm($ID, $options=array()) {
          $showuserlink = 1;
       }
 
+      if (!$this->isNewItem()) {
+         $options['formtitle'] = sprintf(
+            __('%1$s - ID %2$d'),
+            $this->getTypeName(1),
```
Comment (yllen) [inline inc/change.class.php]: It's not a correct gettext sentence.
You must use a sentence for ID %2$d for language readied from right to left

File: inc/problem.class.php [hunk 2]
```diff
@@ -1055,6 +1055,15 @@ function showForm($ID, $options=array()) {
          $showuserlink = 1;
       }
 
+      if (!$this->isNewItem()) {
+         $options['formtitle'] = sprintf(
+            __('%1$s - ID %2$d'),
+            $this->getTypeName(1),
```
Comment (yllen) [inline inc/problem.class.php]: It's not a correct gettext sentence.

File: inc/ticket.class.php [hunk 3]
```diff
@@ -3958,8 +3958,7 @@ function showForm($ID, $options=array()) {
       echo "<th colspan='4'>";
 
       if ($ID) {
-         $text = sprintf(__('%1$s - %2$s'), $this->getTypeName(1),
-                         sprintf(__('%1$s: %2$s'), __('ID'), $ID));
+         $text = sprintf(__('%1$s - ID %2$d'), $this->getTypeName(1), $ID);
          if ($ismultientities) {
```
Comment (yllen) [inline inc/ticket.class.php]: It's not a correct gettext sentence.

File: inc/change.class.php (hunk 1 above)
Comment (trasher) [reply inc/change.class.php]: So this is alredy warong in some other places (commondbtm, commonglpi, ...)

**Pull request merged**